test/Microsoft.Framework.Runtime.Tests/TestSample/project.lock.sample

```diff
@@ -0,0 +1,2180 @@
+{
```

Comment:
The unit test shouldn't reference to functional test util. I prefer a hardcopy of lock file in the git so that we leave trails whenever the baseline is updated. 

The functional test will surely cover the deserializing a real lock file once the runtime is updated.